tests/analysis/adaptation/test_adaptation.py

```diff
@@ -85,24 +84,36 @@ def test_run_benefit_returns_gdf(
                 _expected[1]
             )
 
-    @pytest.fixture
-    def valid_gdf(self) -> GeoDataFrame:
-        return GeoDataFrame(
-            geometry=[Point(x, 0) for x in range(10)],
-        )
-
     @pytest.fixture(name="mocked_adaptation")
-    def _get_mocked_adaptation_fixture(
-        self, valid_gdf: GeoDataFrame
-    ) -> Iterator[Adaptation]:
+    def _get_mocked_adaptation_fixture(self) -> Iterator[Adaptation]:
         # Mock to avoid complex setup.
         @dataclass
         class MockAdaptationOption:
             id: str
 
+            @property
+            def cost_col(self) -> str:
+                return f"{self.id}_cost"
+
```

Comment:
Why not inherinting from the `AdaptationOption`?

Reply:
Adjusted and simplified